src/routes/issue-trigger-rule/getByDeviceType.ts

```diff
@@ -0,0 +1,37 @@
+import Joi from 'joi';
```

Comment:
see comment above. I think you could do without this file by making the previous one more generic